src/components/nimbleMobleListView/NimbleMobileListView.tsx

```diff
@@ -35,6 +35,7 @@ interface MobileDataCardDetail {
 interface MobileListData {
   id: number;
   mainValue?: string;
+  secondaryValue?: string;
```

Comment:
I've added this optional value to set as the parameter to determine whether to show default or custom actions